### PR TITLE
Typo Fix In guide / english / computer-hardware / rom / index.md

### PR DESCRIPTION
**Pull request by iamrajiv**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

**Comment by iamrajiv** [comment]
@thecodingaviator please review this

File: guide/english/computer-hardware/rom/index.md [hunk 1]
```diff
@@ -3,19 +3,19 @@ title: ROM
...
 
 #### CD-ROM
-The CD-ROM is a type of compact disc storage object that requires the use of an optical laser to read the data off the disc. The devices used to read the data house the lasers and are called CD-ROM Drives. The disc itself is generally created in a factory and once made, is read only. This is in contrast to the CD-R (compact disc recordables) medium which has the additional ability to hold new data.
+The CD-ROM is a type of compact disc storage object that requires the use of an optical laser to read the data of the disc. The devices used to read the data house the lasers and are called CD-ROM Drives. The disc itself is generally created in a factory and once made, is read-only. This is in contrast to the CD-R (compact disc recordable) medium which has the additional ability to hold new data.
```
Comment (Manish-Giri) [inline guide/english/computer-hardware/rom/index.md]:
```suggestion
The CD-ROM is a type of compact disc storage object that requires the use of an optical laser to read the data off the disc. The devices used to read the data house the lasers and are called CD-ROM Drives. The disc itself is generally created in a factory and once made, is read-only. This is in contrast to the CD-R (compact disc recordable) medium which has the additional ability to hold new data.
```

**Review by Manish-Giri — CHANGES_REQUESTED** [review]
> to read the data off the disc

The usage of `off` is fine here. Please revert.

**Comment by iamrajiv** [comment]
@Manish-Giri thanks for correction

**Review by Manish-Giri — APPROVED** [review]
LGTM. Thank You for your contribution.

**Pull request merged**